docs/en/user-guide.rst

```diff
@@ -0,0 +1,186 @@
+User Guide
+==========
+
+This guide helps you to create a Laminas application with Doctrine Mongo
+ODM integration. If you’re new to Laminas, please read the `Laminas
+documentation <https://docs.laminas.dev/>`__ before you continue.
+
+Install Composer
+----------------
+
+install composer via ``curl -s http://getcomposer.org/installer | php``
+(on windows, download http://getcomposer.org/installer and execute it
+with PHP)
+
+Laminas Skeleton Application
+----------------------------
+
+Create a new Skeleton application with Composer:
+
+.. code:: bash
+
+   $ composer create-project laminas/laminas-mvc-skeleton doctrine-odm-tutorial
+
+Grab a coffee and wait for composer to finish installing your new
+Laminas Application. Now you can start up your application with the php
+built-in web server:
+
+.. code:: bash
+
+   $ cd doctrine-odm-tutorial
+   $ php -S 0.0.0.0:8080 -t public/ public/index.php
+
+For detailed instructions on installing the Laminas MVC Skeleton
+Application follow `this
+link <https://github.com/laminas/laminas-mvc-skeleton>`__.
+
+Install Doctrine Mongo ODM Module
+---------------------------------
+
+Install DoctrineODMModule :
+
+.. code:: bash
+
+   $ composer require doctrine/doctrine-mongo-odm-module
+
+Open ``doctrine-odm-tutorial/configs/application.config.php`` in your
```

Comment:
doctrine-odm-tutorial?  That sounds like a very old reference and is certainly out of scope for Laminas.  Just reference `config/application.config.php`